frontend/src/components/c2-event/index.tsx

```diff
@@ -0,0 +1,259 @@
+// Copyright 2020, Verizon Media
+// Licensed under the terms of the MIT. See LICENSE file in project root for terms.
+
+import * as React from 'react'
+import { default as Input, SharedProps} from 'src/components/input'
+import { C2Event } from 'src/global_types'
+import ComboBox from 'src/components/combobox'
+import { useAsyncComponent } from 'src/helpers'
+import LoadingSpinner from 'src/components/loading_spinner'
+import WithLabel from 'src/components/with_label'
+import classnames from 'classnames/bind'
+const cx = classnames.bind(require('./stylesheet'))
+
+
+export const C2EventTextArea = React.forwardRef((props: SharedProps & {
```

Comment:
Why did you decide to create a text area here, rather than reuse the one from `components/input` ?

Reply:
Thats a very good question. I missed that component entirely when I was writing this initially. I've gone ahead and removed the custom TextArea.